src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -204,6 +204,30 @@ static Activity watching(@Nonnull String name)
         return EntityBuilder.createActivity(name, null, ActivityType.WATCHING);
     }
 
+    /**
+     * Creates a new Activity instance with the specified name.
+     * <br>This will display as {@code Competing in name} in the official client
+     * 
+     * @param  name
+     *         The not-null name of the newly created game
+     * 
+     * @throws IllegalArgumentException
+     *         If the specified name is null, empty, blank or longer than 128 characters
+     * 
+     * @return A valid Activity instance with the provided name with {@link net.dv8tion.jda.api.entities.Activity.ActivityType#COMPETING}
+     *
+     * @incubating This feature is not yet confirmed for the official bot API
+     */
+    @Nonnull
+    @Incubating
+    static Activity competing(@Nonnull String name)
+    {
+        Checks.notBlank(name, "Name");
+        name = name.trim();
+        Checks.check(name.length() <= 128, "Name must not be greater than 128 characters in lentgh");
```

Comment:
Typo at the very end. It should be 'length'.

Reply:
corrected. Thanks